panel/src/components/Dropdowns/DropdownContent.vue

```diff
@@ -17,10 +17,10 @@
 			<!-- @slot Content of the dropdown which overrides passed `options` prop -->
 			<slot>
 				<template v-for="(option, index) in items">
-					<hr v-if="option === '-'" :key="_uid + '-item-' + index" />
+					<hr v-if="option === '-'" :key="id + '-item-' + index" />
```

Comment:
Is this really needed here at all? Do keys need to be globally unique or just within a component? Because otherwise `'item-' + index'` seems totally fine to me.

Reply:
Hmmm actually I'd think just unique within the loop

Reply:
I think so too. I honestly believe that this is from a time when I still thought they need to be globally unique. But I never ran into issues with keys that are just unique within the loop ever since.

Reply:
I'll change it